ikarus/controlroutines/loadcontrol.inl

```diff
@@ -11,35 +11,28 @@
 
 namespace Ikarus {
 template <typename NLS>
-ControlInformation LoadControl<NLS>::run() {
-  ControlInformation info({false});
-  auto& nonOp = nonLinearSolver_->nonLinearOperator();
-  this->notify(ControlMessages::CONTROL_STARTED, static_cast<std::string>(this->name()));
+[[nodiscard(
+    "The run method returns information of the control routine. You should store this information and check if "
+    "it was successful")]] ControlState
+LoadControl<NLS>::run() {
+  ControlState controlState{.success = false, .currentStep = 0, .stepSize = stepSize_, .name = this->name()};
+  auto& nonOp         = nonLinearSolver_->nonLinearOperator();
+  controlState.sol    = &nonOp.firstParameter();
+  controlState.lambda = nonOp.lastParameter();
+  this->notify(ControlMessages::CONTROL_STARTED, controlState);
   auto& loadParameter = nonOp.lastParameter();
 
-  loadParameter = 0.0;
-  this->notify(ControlMessages::STEP_STARTED, 0, stepSize_);
-  auto solverInfo = nonLinearSolver_->solve();
```

Comment:
Since we removed the solve step here, do you think it is nice to check if the norm of the residuum is zero at this state, before any increment happens? Or is it a hard constraint that the input state to a control routine should be in equilibrium?